client/index.html

```diff
@@ -12,15 +12,14 @@
     </div>
     <script src="public/bundle.js" type="text/javascript"></script>
     <script>
-    (function(d, s, id) {
-      var js, fjs = d.getElementsByTagName(s)[0];
-      if (d.getElementById(id)) return;
-      js = d.createElement(s); js.id = id;
-      js.src = `//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.8&appId=1871157256463548`;
-      fjs.parentNode.insertBefore(js, fjs);
-    }(document, 'script', 'facebook-jssdk'));
-    </script>
+      (function(d, s, id) {
+        var js, fjs = d.getElementsByTagName(s)[0];
+        if (d.getElementById(id)) return;
+        js = d.createElement(s); js.id = id;
+        js.src = `//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.8&appId=1871157256463548`;
```

Comment:
For some reason it works if we add the test app url to only the localhost FB_AppID. This didn't work 3 weeks ago...